azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+# We build using a Docker container generated by GafferHQ/build
+container: gafferhq/build:0.0.0
```

Comment:
We should probably tag this in the `build` repo too. Might be nice if we can get that version lock stuff in there though, otherwise its a bit moot. Then we could maybe do `1.0.0` as the `gcc 4.x` legacy build environment, `2.0.0` as the `gcc 6.3` etc? 

Note to self : Don't mess around with dockerhub tags any more :)